CGAL_Core/include/CGAL/CORE/BigFloat.h

```diff
@@ -614,11 +614,23 @@ inline BigFloat gcd(const BigFloat& a, const BigFloat& b) {
   //mpz_tdiv_qr(q.get_mp(), r.get_mp(), a.get_mp(), b.get_mp());
 //}//
 
-
+/* AF
```

Comment:
We need a construction of a `BigRat` from a `BigFloat`  [here ](https://github.com/CGAL/cgal/blob/d979e070bee8f49344125f58e298be51eba84838/Arrangement_on_surface_2/include/CGAL/Arr_rat_arc/Algebraic_point_2.h#L229)in  the class `Algebraic_point_2`